.vscode/launch.json

```diff
@@ -7,11 +7,23 @@
         {
             "type": "node",
             "request": "launch",
-            "name": "Launch Program",
+            "name": "Start Server",
```

Comment:
[VSCode docs](https://code.visualstudio.com/docs/editor/debugging) says:
> In VS Code, there are two core debugging modes, **Launch** and **Attach**

 I also noticed that most other projects start each configuration name with `Launch`. Perhaps it would be best to stay consistent and use `Launch Server` here, and `Attach to Chrome` below.